generate.sh

```diff
@@ -127,9 +122,13 @@ if [ "${AUTOAPI_JAVA:-false}" == "true" ]; then
   javasphinx-apidoc -o source/ $ROOT_DIR/$AUTOAPI_JAVA_PATH/
 fi
 
+if [ -n "$SUBPROJECT_URLS" ]; then
+  source $BASE/multiple_generate.sh
```

Comment:
I tested this with travis. Build log: https://travis-ci.org/pri22296/testYaydoc2/builds/250469570